wot-wg-2023-draft.html

```diff
@@ -210,38 +210,50 @@ <h2>Scope</h2>
       <p>During this charter, the Working Group would work on the following:</p>
 
         <dl>
-          <dt><strong>Update Existing Specifications</strong>:</dt>
-          <dd>The WG will add new features and address issues discovered during implementation of already published specifications. 
-            Unlike the last charter, these would not be limited to backward-compatible updates.
+          <dt>
+		<strong>Support New Use Cases</strong>:
+          </dt>
+          <dd>Address the requirements of new use cases, such as Digital Twins, scalable events, 
```

Comment:
We have talked way too many times about the notion of scalable events. There is still no agreement on what is really meant here. If we are talking about cloud events, it is already proposed for payload binding.

Reply:
I think we can add more use cases, but "scalable events" is a feature, not a use case.  I would be ok with adding "such as Digital Twins and interworking with edge and cloud systems."

I think that "scalable events" are a work item that relates to "Cloud interworking".  However, "integration" might be a better word in general than "interworking".
```suggestion
          <dd>Address the requirements of new use cases, such as Digital Twins and integration with edge and cloud systems.
```